app/models/application_setting.rb

```diff
@@ -16,6 +16,22 @@ class ApplicationSetting
                 arr
               end
 
+  @keys = [
+    'site.display_login_link',
+    'site.only_admin_can_upload',
+    'site.signup_enabled',
+    'site.name',
+    'site.header_inverse',
+    'custom_content.center_top',
+    'custom_content.center_bottom',
+    'custom_content.right_top',
+    'custom_content.header_menus',
+    'custom_content.css',
+    'site.favicon',
+    'site.footer',
+    'site.theme'
```

Comment:
__[RuboCop]__

Style/TrailingCommaInArrayLiteral: Put a comma after the last item of a multiline array.


<!-- Comment by [Sider](https://sider.review) -->